kore/src/Kore/Log/WarnIfLowProductivity.hs

```diff
@@ -25,9 +25,12 @@ newtype WarnIfLowProductivity =
 
 instance Pretty WarnIfLowProductivity where
     pretty (WarnIfLowProductivity productivityPercent) =
-        Pretty.hsep
-            [ "Warning! Poor performance: productivity dropped to aprox."
-            , Pretty.pretty productivityPercent <> "%"
+        Pretty.vsep
+            [ Pretty.hsep
+                [ "Warning! Poor performance: productivity dropped to aprox."
+                , Pretty.pretty productivityPercent <> "%."
+                ]
+            , "Please file a bug report."
```

Comment:
I want to include the URL so that the user can click on it in the terminal and go directly there:

```suggestion
            [ Pretty.hsep
                [ "Productivity dropped to:"
                , Pretty.pretty productivityPercent <> "%"
                ]
            , "Poor productivity may indicate a performance bug."
            , "Please file a bug report: https://github.com/kframework/kore/issues"
```

I removed the `Warning!` because that is already included by setting the log level below.